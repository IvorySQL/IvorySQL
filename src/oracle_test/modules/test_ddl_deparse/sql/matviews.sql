--
-- Materialized views
--

CREATE MATERIALIZED VIEW ddl_deparse_mv AS
  SELECT * FROM datatype_table LIMIT 1 WITH NO DATA;

REFRESH MATERIALIZED VIEW ddl_deparse_mv;
