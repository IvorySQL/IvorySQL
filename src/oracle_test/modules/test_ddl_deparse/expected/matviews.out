--
-- Materialized views
--
CREATE MATERIALIZED VIEW ddl_deparse_mv AS
  SELECT * FROM datatype_table LIMIT 1 WITH NO DATA;
NOTICE:  DDL test: type simple, tag CREATE MATERIALIZED VIEW
REFRESH MATERIALIZED VIEW ddl_deparse_mv;
NOTICE:  DDL test: type simple, tag REFRESH MATERIALIZED VIEW
